slam/feature_extractor/classical/methods.py

```diff
@@ -0,0 +1,54 @@
+import cv2
```

Comment:
instead of putting it in a dir and using methods.py, create a file named classical.py and put all the methods there.